packages/x-charts/src/ChartsTooltip/index.ts

```diff
@@ -6,3 +6,5 @@ export * from './ChartsItemTooltipContent';
 
 export * from './DefaultChartsAxisTooltipContent';
```

Comment:
Is this correct? I would expect from a public component not to have default.

```suggestion
export * from './ChartsAxisTooltipContent';
```